src/grammar.coffee

```diff
@@ -142,7 +142,7 @@ grammar =
   # they can also serve as keys in object literals.
   AlphaNumeric: [
     o 'NUMBER',                                 -> new NumberLiteral $1
-    o 'INFINITY',                               -> new InfinityLiteral $1
+    o 'INFINITE_NUMBER',                        -> new InfiniteNumberLiteral $1
```

Comment:
I liked the old names. Let's not distinguish between `Infinity` and `2e308`. They should each be an `InfinityLiteral`.